BikeData/src/main/scala/by/itechart/internship/Main.scala

```diff
@@ -0,0 +1,28 @@
+package by.itechart.internship
+
+import com.typesafe.config.ConfigFactory
+
+import scala.util.{Failure, Success, Try}
+
+object Main extends App {
+  val config = ConfigFactory.load("lightbend.conf")
+  val configValues = ConfigValues(config.getString("url.pathFilesStats"),
```

Comment:
Please separate parsing of config from Main class. It's really different responsibility. But the idea with ConfigValues is right